### DISCUSSION TITLE
Outline

### DISCUSSION THREAD

**Discussion by BettyFabre** [huggingface/computer-vision-course#84] [12. Ethics and Biases]
copy - pasted from @snehilsanyal [Draft Outline](https://github.com/johko/computer-vision-course/issues/51)

Hey fellow CV Course Contributors and Reviewers 🤗

This issue discusses an initial draft for the unit Ethics and Bias. We read a few posts, blogs searched through datasets, and created this simple and brief outline. Please feel free to share your views on this, for us to improve this unit.

We prepare this unit by combining theoretical concepts, case studies, and practical examples, and finally close with HuggingFace's mission and efforts to emphasize Ethical AI for Society. The structure is slightly inspired by HF Audio Course.

---- 

**1. Introduction** - Assigned to @snehilsanyal

The [ImageNet Roullete](https://www.chiark.greenend.org.uk/~ijackson/2019/ImageNet-Roulette-cambridge-2017.html) Case Study and Implications

**2. Ethics and Bias in AI** - Assigned to @Zekrom-7780

What is Ethics and Bias? Why does it matter? Include short examples and reflect on previous ImageNet Roullete example.

**3. How bias creeps in AI Models (text, vision, speech)** - Assigned to @Zekrom-7780

Give one example each for each modality with implications and mention the HF space: https://huggingface.co/spaces/society-ethics/

**4. Types of bias in AI** - Assigned to @Zekrom-7780

**5. Bias evaluation in CV models and Metrics** - Assigned to @BettyFabre 

Some example case studies:

[AI recognizes Gay Face](https://scatter.wordpress.com/2017/09/10/guest-post-artificial-intelligence-discovers-gayface-sigh/)
[Twitter Saliency Algorithm Bias](https://www.theregister.com/2021/08/11/defcon_twitter_ai/)
[Fair Diffusion](https://github.com/johko/computer-vision-course/issues/51)
We can also include other studies that discriminate based on gender, ethnicity and other factors.

Example citing/mention, use learnings from the blog: Bias in Text to Image models https://huggingface.co/blog/ethics-soc-4

**6. Bias mitigation in CV models**

**7. HuggingFace's efforts: Ethics and Society** - Assigned to @snehilsanyal

This will be closing chapter of the unit, so we include efforts of HuggingFace in emphasizing Ethics for Society :D

Talk about HuggingFace's mission for transparency and reproducibility (model-cards, datasets, evaluate), other aspects

6 categories of submission of HF Spaces: Rigorous, Consentful, Socially Conscious, Sustainable, Inclusive and Inquisitive
https://huggingface.co/spaces/society-ethics/about

**References:**

[Ethics and Society Letter HuggingFace](https://huggingface.co/blog/ethics-soc-1) 1 to 5
[HuggingFace Audio Course for Structure](https://huggingface.co/learn/audio-course) (great flow of theory, practical examples, and notebooks)
[Ethics course Fast.ai](https://ethics.fast.ai/)
[Kaggle microcourse on Introduction to AI Ethics](https://www.kaggle.com/learn/intro-to-ai-ethics)
[Montreal Ethics AI for Computer Vision](https://montrealethics.ai/the-state-of-the-debate-on-the-ethics-of-computer-vision/)
[Ethical Dimensions of Computer Vision Datasets](https://visualai.princeton.edu/fcvd/assets/Denton_slides.pdf)
Please let us know about the content, suggestions are highly welcome 🤗 🚀 🔥

**Comment by ATaylorAerospace** [comment]
@BettyFabre for Section 5 or 6 and or the Reference Section you might want to include Googles Extensive Skin Tone Research..see below:

https://skintone.google/recommended-practices
